plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2390,15 +2390,40 @@ public int compare(final DiskTO arg0, final DiskTO arg1) {
                 if (volumeObjectTO.getBytesReadRate() != null && volumeObjectTO.getBytesReadRate() > 0) {
```

Comment:
why?

Reply:
Sounds good! The method is getting too big, I don't see any problem with extracting those lines. Additionally, I think that Rafael is suggesting to avoid those duplicated lines, if possible.

Reply:
> why?

In addition to what @GabrielBrascher said, I have probably replied to people demonstrating technical arguments of shorter, well documented, and unit tested methods; and that we need to start this movement, otherwise things get worse, and worse... We have classes with thousands of lines, and that is very unhealthy in the long run.

I am starting to think that I might be the mistaken one. So, I think it is time for me to do the opposite then, I might learn something. Why replicate code around our code base? I mean, what are the technical benefits of replicating code, and not unit testing and documenting them?

Reply:
I can break this out into a method, but the functionality here should already be covered by the tests I added to the LibvirtVMDefTest.java.  What what a test look like that actually adds value here?